.travis.yml

```diff
@@ -145,9 +151,15 @@ install:
     fi
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
-  - conda install -c conda-forge xtl xsimd xtensor
+  - conda install -c conda-forge mamba
+  - if [[ "$HIGHFIVE_USE_XTENSOR" == "True" ]]; then
+      mamba install -c conda-forge xtl xsimd xtensor;
```

Comment:
Just for my curiosity, what's this `mamba`, and why is it needed?

Reply:
It is a drop-in replacement of `conda` (see: https://github.com/TheSnakePit/mamba ). Contrary to `conda` it is implemented in C++, and therefore simply much faster. So there is no need, but it simply renders the CI faster

Reply:
So we just install another package manager with the first package manager that is already replacing the canonical python package manager?

Reply:
Well I'm not sure that I'd call conda a replacement for the canonical python package manager. Also `mamba` is not so much replacing `conda`, as it is accelerating resolving and downloading of packages.

Reply:
I'm being a bit sarcastic here; But wouldn't it be better if these `mamba` improvements happened in `conda`'s core rather than tacking on yet another executable?

Also, how does this "faster download" quantify in terms of speed-up here, since we're only seemingly getting 3 packages plus some dependencies?

Reply:
That being said, I'll not stand between these changes and `master`, I just have questions ;)

Reply:
Thanks!